plugins/flutter_aepedge/android/src/main/java/com/adobe/marketing/mobile/flutter/flutter_aepedge/FlutterAEPEdgeDataBridge.java

```diff
@@ -50,23 +52,36 @@ protected static ExperienceEvent eventFromMap(final Map map) {
              return null;
          }
 
-        Map<String, Object> xdmData = getNullableMap(map, XDM_DATA_KEY);
         String datasetId = null;
+        String datastreamIdOverride = null;
```

Comment:
Why are the declarations in two places? One here and the others after xdmData null check?